tests/test_merge_db_functions.py

```diff
@@ -0,0 +1,337 @@
+          'miopenConvolutionFwdAlgoImplicitGEMM':
+              'ConvHipImplicitGemmForwardV4R4Xdlops_Padded_Gemm,0.02512,0,miopenConvolutionFwdAlgoImplicitGEMM,not used',
+          'miopenConvolutionFwdAlgoDirect':
+              'ConvOclDirectFwd,0.02544,0,miopenConvolutionFwdAlgoDirect,not used',
+          'miopenConvolutionFwdAlgoWinograd':
+              'ConvBinWinogradRxSf2x3g1,0.03104,0,miopenConvolutionFwdAlgoWinograd,not used',
+          'miopenConvolutionFwdAlgoGEMM':
+              'GemmFwdRest,0.06096,414720,miopenConvolutionFwdAlgoGEMM,not used'
+      }
+  }
+
+  key = '1-160-698-5x5-64-79-348-1-1x1-2x2-1x1-0-NCHW-FP32-F'
+  local_paths = ['/data/tests/../utils/test_files/usr_gfx90a68.HIP.fdb.txt']
+  mids = [-1]
+  keep_keys = False
+  update_master_list(master_list, local_paths, mids, keep_keys)
+
+
+def test_write_merge_results():
+
+  master_list = {
+      '1-48-480-3x3-16-48-480-1-1x1-1x1-1x1-0-NCHW-FP16-F': {
+          'miopenConvolutionFwdAlgoImplicitGEMM':
+              'ConvHipImplicitGemmForwardV4R4Xdlops_Padded_Gemm,0.02512,0,miopenConvolutionFwdAlgoImplicitGEMM,not used',
+          'miopenConvolutionFwdAlgoDirect':
+              'ConvOclDirectFwd,0.02544,0,miopenConvolutionFwdAlgoDirect,not used',
+          'miopenConvolutionFwdAlgoWinograd':
+              'ConvBinWinogradRxSf2x3g1,0.03104,0,miopenConvolutionFwdAlgoWinograd,not used',
+          'miopenConvolutionFwdAlgoGEMM':
+              'GemmFwdRest,0.06096,414720,miopenConvolutionFwdAlgoGEMM,not used'
+      }
+  }
+
+  final_file = '/data/tests/old_gfx90a68.HIP.fdb.txt'
+  copy_files = []
+
+  write_merge_results(master_list, final_file, copy_files)
+
+
+def test_merge_text_file():
+  master_file = '/data/MITunaX/tests/../utils/test_files/old_gfx90a68.HIP.fdb.txt'
+  copy_only = False
+  keep_keys = False
+  target_file = '/data/tests/../utils/test_files/usr_gfx90a68.HIP.fdb.txt'
+
+  result_file = merge_text_file(master_file, copy_only, keep_keys, target_file)
+
+  if (os.stat(result_file)):
+    assert (True)
+  else:
+    assert (False)
+
+  #targetr_file set to None
+  result_file = merge_text_file(master_file=None,
+                                copy_only=False,
+                                keep_keys=False,
+                                target_file=None)
+
+
+def test_get_sqlite_table():
+
+  local_path = '/data/utils/test_files/test_gfx90678.db'
+  cnx_from = sqlite3.connect(local_path)
+  perf_rows, perf_cols = get_sqlite_table(cnx_from, 'perf_db')
+
+  if not perf_rows:
+    assert (False)
+  else:
+    assert (True)
+
+  if not perf_cols:
+    assert (False)
+  else:
+    assert (True)
+
+
+def test_get_sqlite_row():
+
+  local_path = '/data/utils/test_files/test_gfx90678.db'
+  cnx_from = sqlite3.connect(local_path)
+
+  perf_rows, perf_cols = get_sqlite_table(cnx_from, 'perf_db')
+  for row in perf_rows:
+    perf = dict(zip(perf_cols, row))
+
+    cfg_row, cfg_cols = get_sqlite_row(cnx_from, 'config', perf['config'])
```

Comment:
function imported from analyze_parse_db

Reply:
Yes, You are correct. As it is reffered in merge_db.py, to cover all API , added the test case.